### PR TITLE
Storage swagger update

### PR DESCRIPTION
**Pull request by tg-msft**
There are three commits in this PR that are easier to review individually.

The first commit sorts the generated models alphabetically to make later diffs cleaner.

The second commit changes to the official versions of the Storage swagger and adds README.md transformations to address any deltas.  I've regenerated and the only code change was dropping a few of the RequestId/Version/Date headers that we accidentally missed before.

The third commit updates blobs to support STG69 features, does minimal work to get things running, and updates the recorded tests against 2019-02-02.  Three UserDelegationKey + SAS tests were disabled per #7309.

### CODE REVIEW

**Review by seanmcc-msft — APPROVED** [review]

**Comment by seanmcc-msft** [comment]
Can you enable the echo client ID feature when generating, if you haven't already?

File: sdk/storage/Azure.Storage.Blobs/swagger/readme.md [hunk 1]
```diff
@@ -3,21 +3,1028 @@
...
+```
+
+### /{containerName}?comp=lease&restype=container&acquire
+``` yaml
+directive:
+- from: swagger-document
+  where: $["x-ms-paths"]["/{containerName}?comp=lease&restype=container&acquire"]
+  transform: >
+    $.put.responses["201"].description = "The lease operation completed successfully.";
+    $.put.responses["201"].headers["x-ms-lease-id"].description = "Uniquely identifies a container or blob's lease";
```
Comment (kfarmer-msft) [inline sdk/storage/Azure.Storage.Blobs/swagger/readme.md]: typo:  "a container's or blob's lease"

Comment (kfarmer-msft) [reply sdk/storage/Azure.Storage.Blobs/swagger/readme.md]: .. and elsewhere

File: sdk/storage/Azure.Storage.Blobs/swagger/readme.md [hunk 2]
```diff
@@ -3,21 +3,1028 @@
...
+directive:
+- from: swagger-document
+  where: $.definitions.KeyInfo
+  transform: >
+    $.required = ["Expiry"];
+    $.properties.Start.type = $.properties.Expiry.type = "date-time-8601";
+    $["x-ms-public"] = false;
+```
+
+### Hide *Include types
```
Comment (kfarmer-msft) [inline sdk/storage/Azure.Storage.Blobs/swagger/readme.md]: nit:  this breaks markdown rendering

**Review by kfarmer-msft — CHANGES_REQUESTED** [review]
a typo and a nit

A link at the start of the readmes to how those transforms work would also be useful for maintainers

**Comment by tg-msft** [comment]
@seanmcc-msft  - we've got #6899 tracking that change.  This PR is already pretty massive and that should be done separately.

@kfarmer-msft - I've already got

> ## Customizations for Track 2 Generator
> See the [AutoRest samples](https://github.com/Azure/autorest/tree/master/Samples/3b-custom-transformations)
for more about how we're customizing things.

at the top of each file which is the only documentation I'm aware of for this feature.

**Review by kfarmer-msft — APPROVED** [review]

**Pull request merged**